forecasttools/trajectories_to_quantiles.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import polars as pl
+
+
+def trajectories_to_quantiles(
+    trajectories: pl.DataFrame,
+    quantiles: list[int] = None,
+    timepoint_cols: list[str] = ["timepoint"],
+    value_col: str = "value",
+    quantile_value_name: str = "quantile_value",
+    quantile_level_name: str = "quantile_level",
+    id_cols: list[str] = None,
+) -> pl.DataFrame:
+    # set default quantiles
+    if quantiles is None:
+        quantiles = (
+            [0.01, 0.025] + list(np.arange(0.05, 1.0, 0.05)) + [0.975, 0.99]
```

Comment:
Printing `list(np.arange(0.05, 1.0, 0.05))` yields values like ` np.float64(0.30000000000000004)` rather than `0.3`. Please ensure the value is correct when writing the output in flusight format.

Reply:
I think `[0.05 * x for x in range(1, 20)] would if anything be more readable in this context if we're not going to write out every quantile level